### DISCUSSION TITLE
Saleor build Pytest failing

### DISCUSSION THREAD

**Discussion by bandieramonte** [saleor/saleor#9914] [Deployment] [answered]
Hi all,
I've just created my first PR to push my changes into my fork from the Saleor API (github.com/saleor/saleor).
Automatically some build tests are run when the PR is created. Some of these tests fail for reasons I can't seem to solve. 
Attached herein are the test logs. Can you please help me to find the solution? 2 tests fail, that is, NyanKiyoshi and linting. The latter fails since it cannot find Python 3.10. Please note that in my pre-commit-config.yaml file I'm using Python3.9, though I'm not sure whether this is connected. NyanKiyoshi fails since:
_level=fatal msg="invocation for event 'failed to post diff comment from file: got '500 Internal Server Error': {\"message\":\"Internal Server Error\"}' failed: %!w(MISSING)"_
[logs_23.zip](https://github.com/saleor/saleor/files/8838287/logs_23.zip)

**Comment by karolkielecki** [comment, marked as answer]
Hi @bandieramonte, thanks for reporting! We've merged a fix into the main of our Saleor repo. Check this out: https://github.com/saleor/saleor/pull/9944. Cheers!

**Reply by bandieramonte** [reply]
Hi @karolkielecki , thanks for getting back to me. I'll test this fix from #9944 and get back to you.

**Reply by bandieramonte** [reply]
Hi again @karolkielecki . I tested this and your fix works fine. However, still another automated test failed 2 days ago. 
The test is called "Bump-Dependencies", and checking the logs I see it failed while creating a PR. But this is strange, I have not created a new PR since at least a 10 days ago. I've just uploaded the copy of the log file. Error says:

_2022-06-20T08:13:39.7775859Z [command]/usr/bin/git remote prune origin
2022-06-20T08:13:40.7560692Z fatal: could not read Username for 'https://github.com': No such device or address
2022-06-20T08:13:40.7598934Z ##[error]The process '/usr/bin/git' failed with exit code 128_

Any insights onto this issue?

[10_Create Pull Request.txt](https://github.com/saleor/saleor/files/8958747/10_Create.Pull.Request.txt)
